ppa/archive/templatetags/ppa_tags.py

```diff
@@ -36,7 +36,7 @@ def querystring_replace(context, **kwargs):
     for key, val in kwargs.items():
         querystring[key] = val
     # return urlencoded query string
-    return querystring.urlencode()
+    return mark_safe(querystring.urlencode())
```

Comment:
any idea why this wasn' t needed before?

Reply:
It gives us `&amp;` in between the query params in the cluster_work.html template instead of `&`.

Reply:
yes, that seems like a reasonable use of mark safe!  I just don't understand why we need it now when didn't need it before, since we're using this template tag for the pagination and we don't have that problem there

Reply:
Whew thanks for pointing this out! Glad we don't need it after all.

Reply:
me too! thanks for figuring it out